doc/md/guides/orms/hibernate.mdx

```diff
@@ -0,0 +1,439 @@
+Once this happens, the responsibility for planning migration scripts and making
+sure they are in line with what Hibernate expects at runtime is moved to developers.
+
+Atlas can automatically plan database schema migrations for developers using Hibernate.
+Atlas plans migrations by calculating the diff between the _current_ state of the database,
+and its _desired_ state.
+
+In the context of versioned migrations, the current state can be thought of as the database schema that would have been created by applying
+all previous migration scripts.
+
+The desired schema of your application can be provided to Atlas via an [External Schema Datasource](/atlas-schema/projects#data-source-external_schema)
+which is any program that can output a SQL schema definition to stdout.
+
+To use Atlas with Hibernate, users can utilize the [Hibernate Atlas Provider](https://github.com/ariga/atlas-provider-hibernate),
+a small program that can be used to load the schema of a Hibernate project into Atlas.
+
+In this guide, we will show how Atlas can be used to automatically plan schema migrations for
+Hibernate users.
+
+## Prerequisites
+
+* A local [Hibernate](https://hibernate.org/orm/) project.
+
+If you don't have a Hibernate project handy, check out [hibernate getting started page](https://docs.jboss.org/hibernate/orm/current/userguide/html_single/Hibernate_User_Guide.html)
+
+## Using the Atlas Hibernate Provider
+
+In this guide, we will use the [Hibernate Atlas Provider](https://github.com/ariga/atlas-provider-hibernate)
+to automatically plan schema migrations for a Hibernate project.
+
+### Installation
+
+Install Atlas from macOS or Linux by running:
+```bash
+curl -sSf https://atlasgo.sh | sh
+```
+See [atlasgo.io](https://atlasgo.io/getting-started#installation) for more installation options.
+
+Install the provider by adding this plugin to your gradle project:
+
+<Tabs>
+<TabItem value="Gradle Kotlin" label="Gradle Kotlin" default>
+
+```gradle
+plugins {
+    id("io.atlasgo.hibernate-provider") version "0.1"
+}
+```
+</TabItem>
+<TabItem value="Gradle Groovy" label="Gradle Groovy" default>
+
+```gradle
+plugins {
+    id "io.atlasgo.hibernate-provider" version "0.1"
+}
+```
+</TabItem>
+<TabItem value="Maven" label="Maven">
+
+```
+Coming soon!
+```
+</TabItem>
+</Tabs>
+
+Verify that the plugin is installed, run this command
+
+<Tabs>
+<TabItem value="Gradle" label="Gradle" default>
+
+```bash
+./gradlew help --task schema
+```
+</TabItem>
+<TabItem value="Maven" label="Maven" default>
+
+```
+Coming soon!
+```
+</TabItem>
+</Tabs>
+
+### Library vs Gradle task
+
+The Atlas Hibernate Provider can be used in two modes:
+* **Gradle Task** - In most cases, you can use the provider as an gradle task to output
+the Hibernate schema and load the schema into Atlas.
+* **Library** - For other case, if you are initializing Hibernate in a custom way,
+using an older version of Hibernate or if you are not using gradle,
```

Comment:
```suggestion
using an older version of Hibernate or if you are not using Gradle,
```

Reply:
I think I would change this to:
'If you are initializing Hibernate in a custom way, using on older version of Hibernate, or if you are not using Gradle you can use the provider....'
And just drop the 'other cases', because it sounds like any other case is niche enough and the user will know if this is what they need